src/riak_network.c

```
@@ -96,7 +108,7 @@
             char ip[INET6_ADDRSTRLEN];
             riak_uint16_t port;
             riak_print_host(addrinfo, ip, sizeof(ip), &port);
-            EVUTIL_CLOSESOCKET(sock);
+            close(sock);
```

It would probably be a good idea.  Maybe we could at least return the error status to the caller.  That said, if the `close()` fails, I'm not sure what recourse you have.  Retrying?